### ISSUE TITLE
message previews should probably tooltip

### ISSUE THREAD

**Issue by t3chguy** [vector-im/element-web#16922]
as quite often very little of a message can be seen in the preview itself, its sometimes useful to peek at more of it, using a native tooltip here should be low risk enough as it only appears after a longer hover
